crates/bins/Cargo.toml

```diff
@@ -51,8 +51,9 @@ rayon = "1.7.0"
 rocket = { version = "=0.5.0", features = ["json"] }
 tracing-subscriber = { version = "0.3.18", features = ["fmt", "env-filter"] }
 thiserror = "1"
-# Temporary integration test
+# Secret Scanning
 secrets = { path = "../secrets", optional = true, default-features = false }
+threadpool = { version = "1.8.1", optional = true }
```

Comment:
Why using this? We already have everything working with `rayon`?

Reply:
Yes but it's adding more dependencies when we do not really have proof is slows things down *significantly*. Can you do without?